src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/InternalUserV6.java

```diff
@@ -79,6 +79,12 @@ public String getHash() {
         public void setHash(String hash) {
             this.hash = hash;
         }
+        /**
+         * Method that gets invoked when "password" field is present in serialized JSON.
+         * Serialized JSON is expected not to have "password" field or have an empty password.
+         * Hence, password field should not be set in InternalUserV6 object in deserialization.
+         */
+        public void setPassword(String password){}
```

Comment:
Any tests?

Reply:
Test that deserialization is successful even with a JSON containing `password` field. I believe it is currently throwing an exception.

Reply:
Done

Reply:
Added tests to test the entire migration process with internal users with empty passwords.